common/test/db_cache/bok_choy_migrations_data_default.sql

```diff
@@ -1,8 +1,8 @@
--- MySQL dump 10.13  Distrib 5.6.30, for Linux (x86_64)
+-- MySQL dump 10.13  Distrib 5.6.31, for Linux (x86_64)
```

Comment:
Note to reviewers: these db_cache bok_choy files are generated by a command. They should be kept up-to-date with our django models to make bok choy runs faster (at least that is my limited understanding!).

I got the commands to update these files from https://openedx.atlassian.net/wiki/display/~zoldak/How+to+do+common+administrative+stuff